core/templates/dev/head/pages/topic_viewer/stories_list_directive.html

```diff
@@ -0,0 +1,26 @@
+<div ng-if="canonicalStoryList.length === 0">
+<em>There are no stories to display.</em>
+</div>
+
+<md-card ng-repeat="canonicalStory in canonicalStories" class="oppia-story-summary-tile">
```

Comment:
Above you have canonicalStoryList, but here canonicalStories?
Also, these will change based on my previous comment of using '&' in the directive.

Reply:
Above was a mistake. Fixed it.